src/tests/actions/articleActions.test.js

```diff
@@ -66,4 +96,65 @@ describe('articleActions', () => {
       ],
     );
   });
+
+  it('should return user artciles', async () => {
+    const response_data = {
+      article: {
+        article: [
+          { author: 10, title: 'React Redux' },
+        ],
+      },
+    };
+
+    mock.onGet('/api/article/my-articles/')
+      .reply(200, response_data);
+    fetchUserArticles()(store.dispatch);
+
+    await flushAllPromises();
+    expect(store.getActions()).toEqual(
+      [
+        { type: GET_USER_ARTICLES_SUCCESS, payload: response_data.article },
+      ],
+    );
+  });
+
+  it('should update an article', async () => {
+    const res_data = {
+      article: {
+        title: 'This is Andela',
+        description: 'I like it',
+        body: 'Come and join us',
+      },
+    };
+    const slug = 'this-is-andela';
+    mock.onPut(`api/articles/${slug}`).reply(200);
+    updateArticle()(store.dispatch);
+    await flushAllPromises();
+    expect(store.getActions()).toEqual(
+      [
+        { type: EDIT_ARTICLE_INITIATED, payload: true },
+        { type: EDIT_ARTICLE_SUCCESS, payload: true },
+      ],
+    );
+  });
+
+  // it('should not update an article if not the author', async () => {
```

Comment:
Please remove the commented block of code

Reply:
@MuhanguziDavid Commented code has been removed from the PR